packages/provider/src/transports/base-wallet-transport.ts

```diff
@@ -206,11 +206,12 @@ export abstract class BaseWalletTransport implements WalletTransport {
     })
   }
 
-  notifyAccountsChanged(accounts: string[]) {
+  notifyAccountsChanged(accounts: string[], origin?: string) {
     this.sendMessage({
       idx: -1,
       type: EventType.ACCOUNTS_CHANGED,
-      data: accounts
+      data: accounts,
```

Comment:
we may want to return `data: { accounts, origin }` .. thoughts?

Reply:
yeah I tried this one, this one breaks communications with dapps since they accept an array here.

Reply:
Gotcha. Could you open issue for us in our issuer-tracker that we should review this event in the future? When we do release of lazy octopus we’ll want all dapps to upgrade, so we can make the change then for what potentially may be more correct